spec/controllers/repositories_controller_spec.rb

```diff
@@ -12,7 +12,10 @@
   describe "GET #index" do
 
     it "assigns all repositories as @repositories" do
-      repository = create(:repository)
+      team = create(:team)
+      TeamUser.create(team: team, user: user, role: TeamUser.roles["owner"])
+      namespace = create(:namespace, team: team)
+      repository = create(:repository, namespace: namespace)
       get :index, {}, valid_session
```

Comment:
Could you please also create a repository that is not visible by the logged in user?